python/pyspark/sql/connect/plan.py

```diff
@@ -1192,6 +1192,7 @@ def plan(self, session: "SparkConnectClient") -> proto.Relation:
         assert self._child is not None
 
         plan = proto.Relation()
+        plan.common.plan_id = self._child._plan_id
```

Comment:
I'm a bit confused, so we have the `plan_id` for all logical plan proto messages, but we never set it before this PR?